internal/common/expand_into_runfiles.bzl

```diff
@@ -1,55 +0,0 @@
-# Copyright 2017 The Bazel Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-"""Helper functions to expand paths into runfiles
-"""
-
-def expand_location_into_runfiles(ctx, path):
-  """Expand a path into runfiles if it contains a $(location).
-
-  If the path has a location expansion, expand it. Otherwise return as-is.
-
-  Args:
-    ctx: context
-    path: the path to expand
-
-  Returns:
-    The expanded path or the original path
-  """
-  if path.find('$(location') < 0:
-    return path
-  return expand_path_into_runfiles(ctx, path)
-
-def expand_path_into_runfiles(ctx, path):
```

Comment:
expand_path_into_runfiles is used in rules_typescript so likely need to keep it around as deprecated